src/containers/pages/ProfileScreen.tsx

```diff
@@ -84,13 +84,13 @@ class ProfileScreen extends React.Component<ProfileProps, ProfileStates> {
                     <ButtonWithColor
                         title="已买宝贝"
                         onPress={() => {
-                            this.props.dispatch(NavigationActions.navigate({routeName: 'bought-products'}))
+                            this.props.dispatch(NavigationActions.navigate({routeName: 'boughtProducts'}))
```

Comment:
I have a question for navigate. actually we can pass params  like 
NavigationAction.navigate({
  routerName: xxxx
  params: {...}
});

But I don't know how to use params in the other page...